bodhi/tests/server/services/test_graphql.py

```diff
@@ -15,12 +15,95 @@
 # You should have received a copy of the GNU General Public License
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+from graphene.test import Client
+
 from bodhi.tests.server import base
+from bodhi.server.graphql_schemas import Release
 
 
 class TestGraphQLService(base.BasePyTestCase):
     """This class contains tests for a /graphql endpoint"""
     def test_get(self):
-        """Ensure that "Hello World" is returned"""
-        res = self.app.get('/graphql')
-        assert res.body == b'"Hello World"'
+        """Ensure that a GraphQL response is returned"""
+        res = self.app.get('/graphql', status="*", expect_errors=True)
+        assert res.body == b'{"errors":[{"message":"Must provide query string."}]}'
+
+    def test_allReleases(self):
+        """Testing allReleases"""
+        client = Client(Release)
```

Comment:
So `Release` here isn't actually your schema. The `schema` you want to pass to your test client here is the collection of queries and mutations you have created. Graphene sqlalchemy actually creates this for you when you call `graphene.Schema(query=Query)`. If you check `graphql.py` you've already done that there so you can use the `schema` variable :)